code/src/main/kotlin/com/expediagroup/sdk/core/http/Status.kt

```diff
@@ -92,10 +94,10 @@ enum class Status(val code: Int) {
     NETWORK_AUTHENTICATION_REQUIRED(511),
 
     // Non-standard status codes (e.g., Apache)
-    THIS_IS_FINE(218); // Non-standard code, used by some Apache modules
+    THIS_IS_FINE(218), // Non-standard code, used by some Apache modules
```

Comment:
Duplicate comment, and I believe the comma is not needed here.

Reply:
Do you think we should ignore the rule enforcing this change? I personally prefer not to, no strong opinion though.